package.json

```diff
@@ -18,7 +18,7 @@
     "node": ">= 8"
   },
   "dependencies": {
-    "@sentry/cli": "^1.70.1"
+    "@sentry/cli": "^1.72.0"
```

Comment:
I'll resolve this properly once the new package is released, and I'll also update the yarn.lock. Leaving here as a placeholder